### PR TITLE
Add Members 04-27-2021

### LINKED ISSUE #2652: REQUEST: New membership for qiiujian16
### GitHub Username

qiujian16

### Organization you are requesting membership in
kubernetes-sigs

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @RainbowMango 
- @pmorie 

### List of contributions to the Kubernetes project
- PRs reviewed / authored
- Issues responded to
- SIG projects I am involved with
[work-api](https://github.com/kubernetes-sigs/work-api)
sig-multicluster

### LINKED ISSUE #2657: REQUEST: New membership for ike-ma
### GitHub Username
@ike-ma

### Organization you are requesting membership in
@kubernetes

### Requirements
- [ Y ] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [ Y ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [ Y ] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [ Y ] I am actively contributing to 1 or more Kubernetes subprojects
- [ Y ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [ Y ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @SergeyKanzhelev 
- @ehashman 

#### PRs reviewed
- #97028 - Adding restart kubelet flag on e2e test 
- kubernetes/test-infra#20106  - Node E2E disruptive job
- #96775 - Kubelet exit on lock contention E2E test
- [WIP] Memory Swap KEP - https://github.com/kubernetes/enhancements/pull/2602#discussion_r616211595  
- Regularly attending SIG-Node meeting, and SIG-CI meeting 
  
#### SIG projects I am involved with
sig-node

### LINKED ISSUE #2658: REQUEST: New membership for BinacsLee
### GitHub Username
@BinacsLee

### Organization you are requesting membership in
@kubernetes
@kubernetes-sigs

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
@damemi 
@Huang-Wei 

### List of contributions to the Kubernetes project
- PRs reviewed / authored
Merged PRs:
https://github.com/kubernetes/kubernetes/pull/101119
https://github.com/kubernetes/kubernetes/pull/101317
https://github.com/kubernetes/kubernetes/pull/101173
https://github.com/kubernetes/kubernetes/pull/101068
https://github.com/kubernetes/kubernetes/pull/101276
https://github.com/kubernetes/kubernetes/pull/101367
https://github.com/kubernetes/kubernetes/pull/101313
https://github.com/kubernetes-sigs/descheduler/pull/554
All of PRs including open and closed:
https://github.com/kubernetes/kubernetes/pulls?q=is%3Apr+author%3ABinacsLee

- SIG projects I am involved with
/sig scheduling

### LINKED ISSUE #2659: REQUEST: New membership for carlisia
### GitHub Username
@carlisia

### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @pi-victor
- @savitharaghunathan

### List of contributions to the Kubernetes project
- Selected as shadow for SIG Docs 1.22 Release Team

### LINKED ISSUE #2660: REQUEST: New membership for chrisnegus
### GitHub Username
@chrisnegus
### Organization you are requesting membership in
@kubernetes

### Requirements
- [x] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [x] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [x] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [x] I am actively contributing to 1 or more Kubernetes subprojects
- [x] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [x] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @pi-victor
- @savitharaghunathan

### List of contributions to the Kubernetes project
- Selected as shadow for SIG Docs 1.22 Release Team

### LINKED ISSUE #2661: REQUEST: New membership for ramrodo
### GitHub Username
@ramrodo

### Organization you are requesting membership in
@kubernetes

### Requirements
- [X] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [X] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [X] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [X] I am actively contributing to 1 or more Kubernetes subprojects
- [X] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [X] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @mkorbi
- @guineveresaenger

### List of contributions to the Kubernetes project
- PRs reviewed / authored:
  - https://github.com/kubernetes/sig-release/pull/1398

- SIG projects I am involved with:
  - CI Signal shadow for Kubernetes 1.22

### LINKED ISSUE #2662: REQUEST: New membership for supriya-premkumar
### GitHub Username
@supriya-premkumar

### Organization you are requesting membership in
@kubernetes

### Requirements
- [ x ] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [ x ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [  x ] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [  x ] I am actively contributing to 1 or more Kubernetes subprojects
- [  x ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [  x ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @dims
-  @savitharaghunathan
- @gsaenger

### List of contributions to the Kubernetes project
- PRs reviewed / authored
   - https://github.com/kubernetes/kubernetes/pull/100801
   - https://github.com/kubernetes/kubernetes/pull/99288
   - https://github.com/kubernetes/test-infra/pull/20934
   - https://github.com/kubernetes/test-infra/pull/21276
- Issues responded to
   - https://github.com/kubernetes/kubernetes/issues/99673
   - https://github.com/kubernetes/kubernetes/issues/80481
   - https://github.com/kubernetes/kubernetes/issues/99243
   - https://github.com/kubernetes/test-infra/issues/18388

- SIG projects I am involved with
   - currently in works with sig-autoscaling
   - https://github.com/kubernetes-sigs/kubetest2/pull/111

### LINKED ISSUE #2663: REQUEST: New membership for gracenng
### GitHub Username
@gracenng

### Organization you are requesting membership in
@kubernetes

### Requirements
- [ x ] I have reviewed the community membership guidelines (https://git.k8s.io/community/community-membership.md)
- [ x ] I have enabled 2FA on my GitHub account (https://github.com/settings/security)
- [ x ] I have subscribed to the kubernetes-dev e-mail list (https://groups.google.com/forum/#!forum/kubernetes-dev)
- [ x ] I am actively contributing to 1 or more Kubernetes subprojects
- [ x ] I have two sponsors that meet the sponsor requirements listed in the community membership guidelines
- [ x ] I have spoken to my sponsors ahead of this application, and they have agreed to sponsor my application

### Sponsors
- @JamesLaverack 
- @guineveresaenger 

### List of contributions to the Kubernetes project
- PRs reviewed / authored
- Issues responded to
- SIG projects I am involved with: sig-release

### PR DESCRIPTION
**Pull request by savitharaghunathan**
Fixes #2663 
Fixes #2662 
Fixes #2661 
Fixes #2660 
Fixes #2659 
Fixes #2658 
Fixes #2652 

/cc @ameukam @palnabarun

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -441,6 +445,7 @@ members:
 - idealhack
 - iheanyi1
 - ihmccreery
+- ike-ma
```
Comment (SergeyKanzhelev) [inline config/kubernetes/org.yaml:448]: @ehashman asked to wait till the first PR here: https://github.com/kubernetes/org/issues/2657 Please delay this membership assignment

Comment (savitharaghunathan) [reply config/kubernetes/org.yaml:448]: Thank you for pointing that out @SergeyKanzhelev. Updating the PR

Comment (savitharaghunathan) [reply config/kubernetes/org.yaml]: Updated the PR

**Review by nikhita — APPROVED** [review]
Thanks @savitharaghunathan! :smile:

/lgtm
/approve

**Pull request merged**